postgres/testcontainers/postgres/__init__.py

```diff
@@ -54,14 +66,24 @@ def __init__(self, image: str = "postgres:latest", port: int = 5432, user: Optio
 
         self.with_exposed_ports(self.port_to_expose)
 
+    @wait_container_is_ready(*ADDITIONAL_TRANSIENT_ERRORS)
+    def _connect(self) -> None:
+        import sqlalchemy
+        engine = sqlalchemy.create_engine(self.get_connection_url(driver=self.DEFAULT_DRIVER))
```

Comment:
Can we just use the `super` call here? It looks like the method body is the same as in the parent class (except for the driver argument which is already handled in the updated `get_connection_url`).

Reply:
@fourteekey, could you please address this question?